docs/explanation/bgp-redirect.rst

```diff
@@ -0,0 +1,91 @@
+========================
+OVN integration with BGP
+========================
```

Comment:
I absolutely love how you have taken the care to properly document this, even though this is for a experimental feature branch. Thank you!

Reply:
How else would I know what I submitted here 2 weeks ago? :laughing: